internal/test/README.md

```diff
@@ -0,0 +1,17 @@
+<!--
+ Copyright 2024 Nutanix. All rights reserved.
+ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Test Framework
```

Comment:
Thanks for this README! How did you copy these, maybe a Makefile target to update these with the current CAPI version would make it easier to maitain?

Reply:
The upstream framework initializes webhooks. I had to remove this imports and initialization code.

I don't think it makes sense to automatically update.

In the future, I'd like upstream  to publish `internal/builder`, so we can remove our copy, and I'll combine `internal/test` and the similar code we already use in `test/helpers`.